src/Containers/SlidingOutPanels.tsx

```diff
@@ -48,91 +49,100 @@ const Image = styled.img`
     width: 400px;
     height: 180px;
     position: absolute;
-    top: 65%;
-    left: 50%;
     border-radius: 15px;
     opacity: 1;
-    transform: translate(-75%, -75%) scale(0.65) perspective(2038px)
-        rotateY(33deg) rotateX(12deg);
-    z-index: 1;
-    animation: move 2s linear infinite;
-    animation-delay: 2s;
 
-    @keyframes move {
-        from {
-            transform: translate(-75%, -75%) scale(0.65) perspective(2038px)
-                rotateY(33deg) rotateX(12deg);
-        }
-        to {
-            transform: translate(-70%, -60%) scale(0.7) perspective(2038px)
+    ${({ id }: { id: string }): {} =>
```

Comment:
better to create an interface to assign meaning to ID and re-use it since duplicate announcement of { id: string}

Reply:
change implemented, handled with custom ts interface props in styled component